src/gasPriceOracle/oracle.ts

```diff
@@ -108,6 +108,7 @@ function _getEthersGasPriceEstimate(
     [CHAIN_IDs.ARBITRUM]: arbitrum.eip1559,
     [CHAIN_IDs.BSC]: ethereum.legacy,
     [CHAIN_IDs.MAINNET]: ethereum.eip1559,
+    [CHAIN_IDs.MONAD]: ethereum.eip1559, // Should we have this? How we determine if we should add chain here?
```

Comment:
eip1559 should be fine for now; if we see an issue we can revert to legacy pricing.